src/components/SuggestedItem/SuggestedItem.css

```diff
@@ -0,0 +1,91 @@
+.item {
+  box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.2),
+    0 1.5px 3.33px 0 rgba(0, 0, 0, 0.19);
+  width: 60px;
+  height: 102px;
+}
+
+.item--border--packed {
```

Comment:
I'm worried this will conflict with the unpacked and packed items. Maybe a better name for this item--border-bundle; A way to distinguish the difference between an item being packed vs bundled.